virtcontainers/device/drivers/vfio.go

```diff
@@ -57,32 +60,29 @@ func (device *VFIODevice) Attach(devReceiver api.DeviceReceiver) error {
 
 	// Pass all devices in iommu group
 	for _, deviceFile := range deviceFiles {
-
 		//Get bdf of device eg 0000:00:1c.0
 		deviceBDF, err := getBDF(deviceFile.Name())
 		if err != nil {
 			return err
 		}
-
```

Comment:
you are not checking here if the device has been already attached here? We should avoid attaching VFIO devices more than once as well.

Reply:
Will modify.